libs/platform/src/lib/components/form/step-input/base.step-input.ts

```diff
@@ -156,7 +156,7 @@ export abstract class StepInputComponent extends BaseInput implements OnInit {
     }
 
     /** @hidden */
-    _contentDensity: ContentDensity = this.config.contentDensity;
+    _contentDensity = this.config.contentDensity;
```

Comment:
why did you remove this?

Reply:
Redundant. But okay, if we're not permitted to change not spec files in spec pr - I will revert this.